addons/website/views/snippets/s_banner_connected.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_banner_connected" name="Banner connected">
+    <section
+        class="s_banner_connected o_cc o_cc5 oe_img_bg pt128 pb136 parallax"
```

Comment:
Is there a specific reason to set the `oe_img_bg` on the `<section/>` itself? As you can see [HERE](https://github.com/odoo/odoo/blob/master/addons/web_editor/static/src/js/editor/snippets.options.js#L7998-L8004), the option's code will automatically consider the `<section/>` as a background element if it has the class leading to break the computing of image data and all options relying on it (image file size, quality...) :grimacing: